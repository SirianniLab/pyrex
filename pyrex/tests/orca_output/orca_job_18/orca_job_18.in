!rhf sto-3g 


*xyz 0 1
C    0.129784282576   -0.143510260831    0.036169063544
O   -1.154576717424   -0.249347260831   -0.030168936456
O    1.080794282576    0.552841739169   -0.032650936456
H    0.341376282576   -1.653519260831    0.337787063544
H   -0.715714717424   -1.454406260831    0.228551063544
*


