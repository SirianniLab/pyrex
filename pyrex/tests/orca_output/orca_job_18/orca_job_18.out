
                                 *****************
                                 * O   R   C   A *
                                 *****************

           --- An Ab Initio, DFT and Semiempirical electronic structure package ---

                  #######################################################
                  #                        -***-                        #
                  #          Department of theory and spectroscopy      #
                  #               Directorship: Frank Neese             #
                  #        Max Planck Institute fuer Kohlenforschung    #
                  #                Kaiser Wilhelm Platz 1               #
                  #                 D-45470 Muelheim/Ruhr               #
                  #                      Germany                        #
                  #                                                     #
                  #                  All rights reserved                #
                  #                        -***-                        #
                  #######################################################


                         Program Version 4.2.0 -  RELEASE  -


 With contributions from (in alphabetic order):
   Daniel Aravena         : Magnetic Suceptibility
   Michael Atanasov       : Ab Initio Ligand Field Theory (pilot matlab implementation)
   Alexander A. Auer      : GIAO ZORA, VPT2
   Ute Becker             : Parallelization
   Giovanni Bistoni       : ED, misc. LED, open-shell LED, HFLED
   Martin Brehm           : Molecular dynamics
   Dmytro Bykov           : SCF Hessian
   Vijay G. Chilkuri      : MRCI spin determinant printing, contributions to CSF-ICE
   Dipayan Datta          : RHF DLPNO-CCSD density
   Achintya Kumar Dutta   : EOM-CC, STEOM-CC
   Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI
   Miquel Garcia          : C-PCM Hessian, Gaussian charge scheme
   Yang Guo               : DLPNO-NEVPT2, CIM, IAO-localization
   Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods
   Benjamin Helmich-Paris : CASSCF linear response (MC-RPA)
   Lee Huntington         : MR-EOM, pCC
   Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3, EOM
   Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density
   Simone Kossmann        : Meta GGA functionals, TD-DFT gradient, OOMP2, MP2 Hessian
   Martin Krupicka        : AUTO-CI
   Lucas Lang             : DCDCAS
   Dagmar Lenk            : GEPOL surface, SMD
   Dimitrios Liakos       : Extrapolation schemes; Compound Job, initial MDCI parallelization
   Dimitrios Manganas     : Further ROCIS development; embedding schemes
   Dimitrios Pantazis     : SARC Basis sets
   Taras Petrenko         : DFT Hessian,TD-DFT gradient, ASA, ECA, R-Raman, ABS, FL, XAS/XES, NRVS
   Peter Pinski           : DLPNO-MP2, DLPNO-MP2 Gradient
   Christoph Reimann      : Effective Core Potentials
   Marius Retegan         : Local ZFS, SOC
   Christoph Riplinger    : Optimizer, TS searches, QM/MM, DLPNO-CCSD(T), (RO)-DLPNO pert. Triples
   Tobias Risthaus        : Range-separated hybrids, TD-DFT gradient, RPA, STAB
   Michael Roemelt        : Original ROCIS implementation
   Masaaki Saitow         : Open-shell DLPNO-CCSD energy and density
   Barbara Sandhoefer     : DKH picture change effects
   Avijit Sen             : IP-ROCIS
   Kantharuban Sivalingam : CASSCF convergence, NEVPT2, FIC-MRCI
   Bernardo de Souza      : ESD, SOC TD-DFT
   Georgi Stoychev        : AutoAux, RI-MP2 NMR
   Willem Van den Heuvel  : Paramagnetic NMR
   Boris Wezisla          : Elementary symmetry handling
   Frank Wennmohs         : Technical directorship


 We gratefully acknowledge several colleagues who have allowed us to
 interface, adapt or use parts of their codes:
   Stefan Grimme, W. Hujo, H. Kruse,             : VdW corrections, initial TS optimization,
                  C. Bannwarth                     DFT functionals, gCP, sTDA/sTD-DF
   Ed Valeev, F. Pavosevic, A. Kumar             : LibInt (2-el integral package), F12 methods
   Garnet Chan, S. Sharma, J. Yang, R. Olivares  : DMRG
   Ulf Ekstrom                                   : XCFun DFT Library
   Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
   Andreas Klamt, Michael Diedenhofen            : otool_cosmo (COSMO solvation model)
   Jiri Pittner, Ondrej Demel                    : Mk-CCSD
   Frank Weinhold                                : gennbo (NPA and NBO analysis)
   Christopher J. Cramer and Donald G. Truhlar   : smd solvation model
   Lars Goerigk                                  : TD-DFT with DH, B97 family of functionals
   V. Asgeirsson, H. Jonsson                     : NEB implementation
   FAccTs GmbH                                   : IRC, NEB, NEB-TS, Multilevel, MM, QM/MM, CI optimization
   S Lehtola, MJT Oliveira, MAL Marques          : LibXC Library


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 Your ORCA version has been built with support for libXC version: 4.2.3
 For citations please refer to: https://tddft.org/programs/libxc/

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines


----- Orbital basis set information -----
Your calculation utilizes the basis: STO-3G
   H-Ne       : W. J. Hehre, R. F. Stewart and J. A. Pople, J. Chem. Phys. 2657 (1969).
   Na-Ar      : W. J. Hehre, R. Ditchfield, R. F. Stewart and J. A. Pople, J. Chem. Phys. 2769 (1970).
   K,Ca,Ga-Kr : W. J. Pietro, B. A. Levy, W. J. Hehre and R. F. Stewart, J. Am. Chem. Soc. 19, 2225 (1980).
   Sc-Zn,Y-Cd : W. J. Pietro and W. J. Hehre, J. Comp. Chem. 4, 241 (1983).

================================================================================
                                        WARNINGS
                       Please study these warnings very carefully!
================================================================================


INFO   : the flag for use of LIBINT has been found!

================================================================================
                                       INPUT FILE
================================================================================
NAME = /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_18/orca_job_18.in
|  1> !rhf sto-3g 
|  2> 
|  3> 
|  4> *xyz 0 1
|  5> C    0.129784282576   -0.143510260831    0.036169063544
|  6> O   -1.154576717424   -0.249347260831   -0.030168936456
|  7> O    1.080794282576    0.552841739169   -0.032650936456
|  8> H    0.341376282576   -1.653519260831    0.337787063544
|  9> H   -0.715714717424   -1.454406260831    0.228551063544
| 10> *
| 11> 
| 12> 
| 13> 
| 14>                          ****END OF INPUT****
================================================================================

                       ****************************
                       * Single Point Calculation *
                       ****************************

---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  C      0.129784   -0.143510    0.036169
  O     -1.154577   -0.249347   -0.030169
  O      1.080794    0.552842   -0.032651
  H      0.341376   -1.653519    0.337787
  H     -0.715715   -1.454406    0.228551

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG     MASS         X           Y           Z
   0 C     6.0000    0    12.011    0.245257   -0.271195    0.068350
   1 O     8.0000    0    15.999   -2.181834   -0.471198   -0.057011
   2 O     8.0000    0    15.999    2.042405    1.044719   -0.061701
   3 H     1.0000    0     1.008    0.645108   -3.124699    0.638325
   4 H     1.0000    0     1.008   -1.352505   -2.748430    0.431899

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 C      0   0   0     0.000000000000     0.00000000     0.00000000
 O      1   0   0     1.290420621012     0.00000000     0.00000000
 O      1   2   0     1.180704162949   147.89259754     0.00000000
 H      1   2   3     1.554307489678    93.77217770   180.02963578
 H      4   1   2     1.081212223731    70.63349888     0.00000000

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 C      0   0   0     0.000000000000     0.00000000     0.00000000
 O      1   0   0     2.438541571278     0.00000000     0.00000000
 O      1   2   0     2.231207513153   147.89259754     0.00000000
 H      1   2   3     2.937215483394    93.77217770   180.02963578
 H      4   1   2     2.043194995500    70.63349888     0.00000000

---------------------
BASIS SET INFORMATION
---------------------
There are 3 groups of distinct atoms

 Group   1 Type C   : 6s3p contracted to 2s1p pattern {33/3}
 Group   2 Type O   : 6s3p contracted to 2s1p pattern {33/3}
 Group   3 Type H   : 3s contracted to 1s pattern {3}

Atom   0C    basis set group =>   1
Atom   1O    basis set group =>   2
Atom   2O    basis set group =>   2
Atom   3H    basis set group =>   3
Atom   4H    basis set group =>   3

Checking for AutoStart:
The File: /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_18/orca_job_18.gbw exists
Trying to determine its content:
     ... Fine, the file contains calculation information
     ... Fine, the calculation information was read
     ... Fine, the file contains a basis set
     ... Fine, the basis set was read
     ... Fine, the file contains a geometry
     ... Fine, the geometry was read
     ... Fine, the file contains a set of orbitals
     ... Fine, the orbitals can be read
     => possible old guess file was deleted
     => GBW file was renamed to GES file
     => GES file is set as startup file
     => Guess is set to MORead
     ... now leaving AutoStart

------------------------------------------------------------------------------
                           ORCA GTO INTEGRAL CALCULATION
------------------------------------------------------------------------------

                         BASIS SET STATISTICS AND STARTUP INFO

 # of primitive gaussian shells          ...   33
 # of primitive gaussian functions       ...   51
 # of contracted shells                  ...   11
 # of contracted basis functions         ...   17
 Highest angular momentum                ...    1
 Maximum contraction depth               ...    3
 Integral package used                   ... LIBINT
 Integral threshhold            Thresh   ...  1.000e-10
 Primitive cut-off              TCut     ...  1.000e-11


------------------------------ INTEGRAL EVALUATION ----------------------------


 * One electron integrals 
 Pre-screening matrix                    ... done
 Shell pair data                         ... done (   0.000 sec)

-------------------------------------------------------------------------------
                                 ORCA SCF
-------------------------------------------------------------------------------

------------
SCF SETTINGS
------------
Hamiltonian:
 Ab initio Hamiltonian  Method          .... Hartree-Fock(GTOs)


General Settings:
 Integral files         IntName         .... /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_18/orca_job_18
 Hartree-Fock type      HFTyp           .... RHF
 Total Charge           Charge          ....    0
 Multiplicity           Mult            ....    1
 Number of Electrons    NEL             ....   24
 Basis Dimension        Dim             ....   17
 Nuclear Repulsion      ENuc            ....     68.6370078498 Eh

Convergence Acceleration:
 DIIS                   CNVDIIS         .... on
   Start iteration      DIISMaxIt       ....    12
   Startup error        DIISStart       ....  0.200000
   # of expansion vecs  DIISMaxEq       ....     5
   Bias factor          DIISBfac        ....   1.050
   Max. coefficient     DIISMaxC        ....  10.000
 Newton-Raphson         CNVNR           .... off
 SOSCF                  CNVSOSCF        .... on
   Start iteration      SOSCFMaxIt      ....   150
   Startup grad/error   SOSCFStart      ....  0.003300
 Level Shifting         CNVShift        .... on
   Level shift para.    LevelShift      ....    0.2500
   Turn off err/grad.   ShiftErr        ....    0.0010
 Zerner damping         CNVZerner       .... off
 Static damping         CNVDamp         .... on
   Fraction old density DampFac         ....    0.7000
   Max. Damping (<1)    DampMax         ....    0.9800
   Min. Damping (>=0)   DampMin         ....    0.0000
   Turn off err/grad.   DampErr         ....    0.1000
 Fernandez-Rico         CNVRico         .... off

SCF Procedure:
 Maximum # iterations   MaxIter         ....   125
 SCF integral mode      SCFMode         .... Direct
   Integral package                     .... LIBINT
 Reset frequency        DirectResetFreq ....    20
 Integral Threshold     Thresh          ....  1.000e-10 Eh
 Primitive CutOff       TCut            ....  1.000e-11 Eh

Convergence Tolerance:
 Convergence Check Mode ConvCheckMode   .... Total+1el-Energy
 Convergence forced     ConvForced      .... 0
 Energy Change          TolE            ....  1.000e-06 Eh
 1-El. energy change                    ....  1.000e-03 Eh
 Orbital Gradient       TolG            ....  5.000e-05
 Orbital Rotation angle TolX            ....  5.000e-05
 DIIS Error             TolErr          ....  1.000e-06


Diagonalization of the overlap matrix:
Smallest eigenvalue                        ... 2.251e-01
Time for diagonalization                   ...    0.003 sec
Threshold for overlap eigenvalues          ... 1.000e-08
Number of eigenvalues below threshold      ... 0
Time for construction of square roots      ...    0.002 sec
Total time needed                          ...    0.005 sec

---------------------
INITIAL GUESS: MOREAD
---------------------
Guess MOs are being read from file: /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_18/orca_job_18.ges
Input Geometry matches current geometry (good)
Input basis set matches current basis set (good)
MOs were renormalized
MOs were reorthogonalized (Cholesky)
                      ------------------
                      INITIAL GUESS DONE (   0.0 sec)
                      ------------------
--------------
SCF ITERATIONS
--------------
ITER       Energy         Delta-E        Max-DP      RMS-DP      [F,P]     Damp
               ***  Starting incremental Fock matrix formation  ***
                      *** Initiating the SOSCF procedure ***
                      *** Re-Reading the Fockian *** 
                      *** Removing any level shift *** 
ITER      Energy       Delta-E        Grad      Rot      Max-DP    RMS-DP
  0   -185.99857486 -185.9985748575  0.000003  0.000003  0.000005  0.000001
               *** Restarting incremental Fock matrix formation ***
                 **** Energy Check signals convergence ****
              ***Rediagonalizing the Fockian in SOSCF/NRSCF***

               *****************************************************
               *                     SUCCESS                       *
               *           SCF CONVERGED AFTER   1 CYCLES          *
               *****************************************************


----------------
TOTAL SCF ENERGY
----------------

Total Energy       :         -185.99857486 Eh           -5061.27853 eV

Components:
Nuclear Repulsion  :           68.63700785 Eh            1867.70794 eV
Electronic Energy  :         -254.63558271 Eh           -6928.98647 eV
One Electron Energy:         -392.54507376 Eh          -10681.69450 eV
Two Electron Energy:          137.90949105 Eh            3752.70804 eV

Virial components:
Potential Energy   :         -370.70338816 Eh          -10087.35202 eV
Kinetic Energy     :          184.70481331 Eh            5026.07349 eV
Virial Ratio       :            2.00700448


---------------
SCF CONVERGENCE
---------------

  Last Energy change         ...   -1.0303e-10  Tolerance :   1.0000e-06
  Last MAX-Density change    ...    6.3313e-06  Tolerance :   1.0000e-05
  Last RMS-Density change    ...    1.0339e-06  Tolerance :   1.0000e-06
  Last Orbital Gradient      ...    2.5020e-06  Tolerance :   5.0000e-05
  Last Orbital Rotation      ...    4.5158e-06  Tolerance :   5.0000e-05

             **** THE GBW FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_18/orca_job_18.gbw) ****
             **** DENSITY FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_18/orca_job_18.scfp) ****
             **** ENERGY FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_18/orca_job_18.en.tmp) ****
             **** THE GBW FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_18/orca_job_18.gbw) ****
             **** DENSITY FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_18/orca_job_18.scfp) ****
----------------
ORBITAL ENERGIES
----------------

  NO   OCC          E(Eh)            E(eV) 
   0   2.0000     -20.369369      -554.2787 
   1   2.0000     -20.272607      -551.6457 
   2   2.0000     -11.267658      -306.6086 
   3   2.0000      -1.428506       -38.8716 
   4   2.0000      -1.330611       -36.2078 
   5   2.0000      -0.724072       -19.7030 
   6   2.0000      -0.674416       -18.3518 
   7   2.0000      -0.595331       -16.1998 
   8   2.0000      -0.574294       -15.6273 
   9   2.0000      -0.452409       -12.3107 
  10   2.0000      -0.366690        -9.9781 
  11   2.0000      -0.307082        -8.3561 
  12   0.0000       0.236325         6.4307 
  13   0.0000       0.279093         7.5945 
  14   0.0000       0.526165        14.3177 
  15   0.0000       0.631006        17.1705 
  16   0.0000       1.089759        29.6538 

                    ********************************
                    * MULLIKEN POPULATION ANALYSIS *
                    ********************************

-----------------------
MULLIKEN ATOMIC CHARGES
-----------------------
   0 C :    0.444943
   1 O :   -0.305545
   2 O :   -0.206794
   3 H :   -0.205538
   4 H :    0.272934
Sum of atomic charges:   -0.0000000

--------------------------------
MULLIKEN REDUCED ORBITAL CHARGES
--------------------------------
  0 C s       :     3.094818  s :     3.094818
      pz      :     0.858253  p :     2.460239
      px      :     0.860861
      py      :     0.741125
  1 O s       :     3.877987  s :     3.877987
      pz      :     1.656469  p :     4.427558
      px      :     1.133456
      py      :     1.637634
  2 O s       :     3.859499  s :     3.859499
      pz      :     1.483403  p :     4.347296
      px      :     1.442087
      py      :     1.421806
  3 H s       :     1.205538  s :     1.205538
  4 H s       :     0.727066  s :     0.727066


                     *******************************
                     * LOEWDIN POPULATION ANALYSIS *
                     *******************************

----------------------
LOEWDIN ATOMIC CHARGES
----------------------
   0 C :    0.367402
   1 O :   -0.278975
   2 O :   -0.134782
   3 H :   -0.204354
   4 H :    0.250710

-------------------------------
LOEWDIN REDUCED ORBITAL CHARGES
-------------------------------
  0 C s       :     3.069711  s :     3.069711
      pz      :     0.864267  p :     2.562887
      px      :     0.927927
      py      :     0.770694
  1 O s       :     3.780611  s :     3.780611
      pz      :     1.654890  p :     4.498365
      px      :     1.196522
      py      :     1.646952
  2 O s       :     3.731314  s :     3.731314
      pz      :     1.480393  p :     4.403468
      px      :     1.474280
      py      :     1.448795
  3 H s       :     1.204354  s :     1.204354
  4 H s       :     0.749290  s :     0.749290


                      *****************************
                      * MAYER POPULATION ANALYSIS *
                      *****************************

  NA   - Mulliken gross atomic population
  ZA   - Total nuclear charge
  QA   - Mulliken gross atomic charge
  VA   - Mayer's total valence
  BVA  - Mayer's bonded valence
  FA   - Mayer's free valence

  ATOM       NA         ZA         QA         VA         BVA        FA
  0 C      5.5551     6.0000     0.4449     3.8716     3.8716    -0.0000
  1 O      8.3055     8.0000    -0.3055     2.1669     2.1669    -0.0000
  2 O      8.2068     8.0000    -0.2068     2.4293     2.4293    -0.0000
  3 H      1.2055     1.0000    -0.2055     0.9578     0.9578    -0.0000
  4 H      0.7271     1.0000     0.2729     0.9255     0.9255    -0.0000

  Mayer bond orders larger than 0.100000
B(  0-C ,  1-O ) :   1.3857 B(  0-C ,  2-O ) :   2.0499 B(  0-C ,  3-H ) :   0.3194 
B(  0-C ,  4-H ) :   0.1166 B(  1-O ,  2-O ) :   0.2253 B(  1-O ,  3-H ) :   0.1987 
B(  1-O ,  4-H ) :   0.3572 B(  3-H ,  4-H ) :   0.3686 

-------
TIMINGS
-------

Total SCF time: 0 days 0 hours 0 min 0 sec 

Total time                  ....       0.433 sec
Sum of individual times     ....       0.130 sec  ( 30.0%)

Fock matrix formation       ....       0.107 sec  ( 24.7%)
Diagonalization             ....       0.000 sec  (  0.0%)
Density matrix formation    ....       0.000 sec  (  0.0%)
Population analysis         ....       0.020 sec  (  4.5%)
Initial guess               ....       0.001 sec  (  0.3%)
Orbital Transformation      ....       0.000 sec  (  0.0%)
Orbital Orthonormalization  ....       0.000 sec  (  0.0%)
DIIS solution               ....       0.000 sec  (  0.0%)
SOSCF solution              ....       0.002 sec  (  0.4%)

-------------------------   --------------------
FINAL SINGLE POINT ENERGY      -185.998574857652
-------------------------   --------------------


                            ***************************************
                            *     ORCA property calculations      *
                            ***************************************

                                    ---------------------
                                    Active property flags
                                    ---------------------
   (+) Dipole Moment


------------------------------------------------------------------------------
                       ORCA ELECTRIC PROPERTIES CALCULATION
------------------------------------------------------------------------------

Dipole Moment Calculation                       ... on
Quadrupole Moment Calculation                   ... off
Polarizability Calculation                      ... off
GBWName                                         ... /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_18/orca_job_18.gbw
Electron density file                           ... /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_18/orca_job_18.scfp
The origin for moment calculation is the CENTER OF MASS  = ( 0.000044, -0.000036  0.000010)

-------------
DIPOLE MOMENT
-------------
                                X             Y             Z
Electronic contribution:      0.24871       2.51460      -0.46184
Nuclear contribution   :     -0.35233      -2.91126       0.53039
                        -----------------------------------------
Total Dipole Moment    :     -0.10362      -0.39666       0.06854
                        -----------------------------------------
Magnitude (a.u.)       :      0.41566
Magnitude (Debye)      :      1.05652



--------------------
Rotational spectrum 
--------------------
 
Rotational constants in cm-1:     3.122233     0.361069     0.323642 
Rotational constants in MHz : 93602.199316 10824.573480  9702.529990 

 Dipole components along the rotational axes: 
x,y,z [a.u.] :    -0.245459    -0.335443    -0.000023 
x,y,z [Debye]:    -0.623908    -0.852629    -0.000058 

 

Timings for individual modules:

Sum of individual times         ...        0.974 sec (=   0.016 min)
GTO integral calculation        ...        0.453 sec (=   0.008 min)  46.5 %
SCF iterations                  ...        0.521 sec (=   0.009 min)  53.5 %
                             ****ORCA TERMINATED NORMALLY****
TOTAL RUN TIME: 0 days 0 hours 0 minutes 1 seconds 307 msec
